brandon-brandonmcastillo/week-3/giffaw/scripts/app.js

```diff
@@ -0,0 +1,35 @@
+$(document).ready(function() {
+    console.log("hey");
+    // Website URL Key Variable
+    var website = 'https://api.giphy.com/v1/gifs/search?q=';
+    // Your API Key Variable
+    var api_key = "Wnie8mwuHu91k4SzgDpUG6dKeEBxMdIQ";
+
+    // Form Click with prevent default
+    $('form').on('submit', function(e) {
+        e.preventDefault();
+
+
+        // Saves Text Input as a value
+        var q = $('.gif-input').val();
+
+        // GET REQUEST
+        $.ajax({
+            method: 'GET',
+            url: website + q + "&api_key=" + api_key,
+            // success function
+            success: function(res) {
+                console.log(res);
+                res.data.map(index => {
```

Comment:
A little tip here. The ```.map``` function returns a duplicate of the array vs looping through the original array. This uses more memory and is not as efficient in this use case as a ```.forEach()``` would be.